state/convert_test.go

```diff
@@ -136,6 +132,8 @@ func TestDeriveEventsHash(t *testing.T) {
 			eventHashes = append(eventHashes, hash)
 		}
 		var execResult *entities.ExecutionResult
+
+		//TODO: ILLIA: chunk[0].EventCollection is empty! but should be equal to hash?
```

Comment:
is this from the live network?

Reply:
Sorry, this is obsolete. I noticed such a behavior only for the root block of a new spork. Is it expected?